docs/api-overview.mdx

```diff
@@ -88,6 +90,11 @@ _More information on the partner UUID use case can be found in the [Issuance Gui
 If <b>mandatory</b> or <b>required</b> fields are both missing, then the credential will be treated as optional.
 :::
 
+:::note Note 3
```

Comment:
Mind dropping this number Note convention? I think we could choose different syntax highlights between the three "notes" to break up the sameness.

Reply:
Also, please mention that `required` in Note 2 is now deprecated.